app/views/homes/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1><%= title.upcase %></h1>
```

Comment:
This is pretty pedantic, but I'd still recommend making it upcase in the stylesheet since uppercase is a stylistic, not infrastructural choice.